### PR TITLE
Added GNDVI and SWI Indices 

### PR DESCRIPTION
**Pull request by MATRIX4284**
Added GNDVI and SWI Indices

### CODE REVIEW

**Review by MATRIX4284 — COMMENTED** [review]
thanks

**Comment by MATRIX4284** [comment]
can this be merged now.

**Comment by adamjstewart** [comment]
This still needs unit tests, you should be able to copy and modify the tests for the other indices, see `tests/transforms/test_indices.py`.

**Comment by calebrob6** [comment]
@adamjstewart the coverage for the various indices are actually provided by test_transforms.py, I'm going to add specific tests in test_indices.py so that it is less confusing/ conforms with the standard way of doing things

**Comment by calebrob6** [comment]
Okay I think this is good to go now

**Comment by MATRIX4284** [comment]
thank you

File: tests/transforms/test_indices.py [hunk 1]
```diff
@@ -7,7 +7,16 @@
...
-from torchgeo.transforms import indices
+from torchgeo.transforms import (
+    AppendGNDVI,
+    AppendNBR,
+    AppendNDBI,
+    AppendNDSI,
+    AppendNDVI,
+    AppendNDWI,
+    AppendSWI,
+    indices,
```
Comment (adamjstewart) [inline tests/transforms/test_indices.py:18]: Could replace this with `AppendNormalizedDifferenceIndex` for consistency

File: torchgeo/transforms/indices.py [hunk 2]
```diff
@@ -165,3 +165,39 @@ def __init__(self, index_green: int, index_nir: int) -> None:
...
+
+    * https://doi.org/10.3390/w13121647
+    """
+
+    def __init__(self, index_red: int, index_swir: int) -> None:
+        """Initialize a new transform instance.
+
+        Args:
+            index_red: index of the VRE1 band, e.g. B5 in Sentinel 2 imagery
+            index_swir: index of the SWIR2 band, e.g. B11 in Sentinel 2 imagery
```
Comment (adamjstewart) [inline torchgeo/transforms/indices.py:183]: We don't add examples for other indices. Should we?

**Review by adamjstewart — DISMISSED** [review]
Looks good, couple comments that are optional to address

**Review by calebrob6 — APPROVED** [review]

File: torchgeo/transforms/indices.py (hunk 2 above)
Comment (calebrob6) [reply torchgeo/transforms/indices.py:183]: Nah I think it is fine

File: tests/transforms/test_indices.py (hunk 1 above)
Comment (calebrob6) [reply tests/transforms/test_indices.py]: It took me a startling long time to figure out what you meant here

**Review by calebrob6 — APPROVED** [review]

**Comment by calebrob6** [comment]
Thanks again for the contribution @MATRIX4284

**Pull request merged**